justega247/Maintenance-Tracker
# #164860286: Admin can manage requests

#### What does this PR do?
- This PR is to enable admin to be able to manage a request. The actions that can be carried out include approve, disapprove and resolve a request
#### Description of Task to be completed?
- Write code to implement the functionality of the endpoint
#### How should this be manually tested?
- Login as the admin and click on the manage button for any of the displayed requests
- You should see a form where you can manage the state of the request.
#### What are the relevant pivotal tracker stories?
[#164860286](https://www.pivotaltracker.com/story/show/164860286)
